pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -489,6 +489,50 @@ func newCertRotationController(
 	)
 	ret.certRotators = append(ret.certRotators, certRotator)
 
+	certRotator = certrotation.NewCertRotationController(
+		"MachineSystemAdminClient",
+		certrotation.SigningRotation{
+			Namespace:              operatorclient.OperatorNamespace,
+			Name:                   "machine-system-admin-signer",
+			Validity:               10 * 365 * defaultRotationDay,
+			Refresh:                8 * 365 * defaultRotationDay,
+			RefreshOnlyWhenExpired: refreshOnlyWhenExpired,
+			Informer:               kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets(),
+			Lister:                 kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets().Lister(),
+			Client:                 kubeClient.CoreV1(),
+			EventRecorder:          eventRecorder,
+		},
+		certrotation.CABundleRotation{
+			Namespace:     operatorclient.OperatorNamespace,
+			Name:          "machine-system-admin-ca",
+			Informer:      kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().ConfigMaps(),
+			Lister:        kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().ConfigMaps().Lister(),
+			Client:        kubeClient.CoreV1(),
+			EventRecorder: eventRecorder,
+		},
+		certrotation.TargetRotation{
+			Namespace: operatorclient.OperatorNamespace,
+			Name:      "machine-system-admin-client",
+			// This needs to be long lived so users can download it as backup kubeconfig if the initial one would be lost
+			Validity:               10 * 365 * defaultRotationDay,
```

Comment:
30 days and 15 days is reasonable I think.

Reply:
same reasons for the client

Reply:
I made it 120 and 30 so it always lives at least 30 days longer then kube-control-plane-signer. Does that work for you?